pages.es/common/git-rebase.md

```diff
@@ -4,9 +4,9 @@
 > Se utiliza comúnmente para "mover" una rama entera a otra base, ya que crea copias de los commits en una nueva ubicación.
 > Más información: <https://git-scm.com/docs/git-rebase>.
 
-- Rebasa la rama actual en lo más alto de la rama master:
+- Rebasa la rama actual en lo más alto de la rama branch_name:
```

Comment:
```suggestion
- Rebasa la rama actual en lo más alto de una rama:
```
Change "the branch branch_name" to "a branch".

Reply:
Thanks! I forgot to mention that the `branch_name` needed translating here, since I don't speak the language haha